components/nav/NavFooter.vue

```diff
@@ -28,68 +22,14 @@ function toggleDark() {
           @click="togglePreferences('zenMode')"
         />
       </CommonTooltip>
-      <CommonTooltip :content="$t('settings.about.sponsor_action')">
-        <NuxtLink
-          flex
-          text-lg
-          i-ri-heart-3-line hover="i-ri-heart-3-fill text-rose"
-          :aria-label="$t('settings.about.sponsor_action')"
-          href="https://github.com/sponsors/elk-zone"
-          target="_blank"
-        />
-      </CommonTooltip>
     </div>
     <div>
-      <i18n-t v-if="isHydrated" keypath="nav.built_at">
-        <time :datetime="String(buildTimeDate)" :title="$d(buildTimeDate, 'long')">{{ buildTimeAgo }}</time>
-      </i18n-t>
-      <span v-else>
-        {{ $t('nav.built_at', [$d(buildTimeDate, 'shortDate')]) }}
-      </span>
-      &middot;
-      <NuxtLink
-        v-if="buildInfo.env === 'release'"
-        external
-        :href="`https://github.com/elk-zone/elk/releases/tag/v${buildInfo.version}`"
-        target="_blank"
-        font-mono
-      >
-        v{{ buildInfo.version }}
-      </NuxtLink>
-      <span v-else>{{ buildInfo.env }}</span>
-      <template v-if="buildInfo.commit && buildInfo.branch !== 'release'">
-        &middot;
-        <NuxtLink
-          external
-          :href="`https://github.com/elk-zone/elk/commit/${buildInfo.commit}`"
-          target="_blank"
-          font-mono
-        >
-          {{ buildInfo.shortCommit }}
-        </NuxtLink>
-      </template>
-    </div>
-    <div>
-      <NuxtLink cursor-pointer hover:underline to="/settings/about">
-        {{ $t('settings.about.label') }}
-      </NuxtLink>
-      <template v-if="config.public.privacyPolicyUrl">
-        &middot;
-        <NuxtLink cursor-pointer hover:underline :to="config.public.privacyPolicyUrl">
-          {{ $t('nav.privacy') }}
-        </NuxtLink>
-      </template>
-      &middot;
-      <NuxtLink href="/m.webtoo.ls/@elk" target="_blank">
-        Mastodon
-      </NuxtLink>
-      &middot;
-      <NuxtLink href="https://chat.elk.zone" target="_blank" external>
-        Discord
+      <NuxtLink href="https://mozilla.social/privacy-policy" target="_blank" external>
+        Privacy Policy
       </NuxtLink>
       &middot;
-      <NuxtLink href="https://github.com/elk-zone/elk" target="_blank" external>
-        GitHub
+      <NuxtLink href="https://mozilla.social/terms" target="_blank" external>
```

Comment:
I wonder if we can make these section "programmable" via the config file. In this way, we can uplift this PR too.